package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "docs",
+  "scripts": {
+    "lint": "yarn install && yarn lint-md && yarn lint-js && yarn lint-php && yarn lint-language",
+    "lint-fix": "yarn install && yarn lint-md --fix; yarn lint-js --fix; yarn lint-php --fix; echo 'Fixed all auto-fixable problems'",
+    "lint-md": "yarn install && markdownlint-cli2 en/**/*.md",
+    "lint-js": "yarn install && eslint en",
```

```suggestion
    "lint": "yarn lint-md && yarn lint-js && yarn lint-php && yarn lint-language",
    "lint-fix": "yarn lint-md --fix; yarn lint-js --fix; yarn lint-php --fix; echo 'Fixed all auto-fixable problems'",
    "lint-md": "markdownlint-cli2 en/**/*.md",
    "lint-js": "eslint en",
```
We don't run `yarn install` in any of our other package.json scripts so shouldn't here either.  There's only a few people who will ever run these scripts and we all have a general understanding that you need to `yarn install` before you do things

---

Done. I'll add some documentation about how to run linting since it's no longer just "run the lint command"

I will note though that none of our other package.json scripts are for this purpose - so we don't _have_ to align with how they're set up. I'm only making this change now to avoid unnecessarily PR ping pong.